lib/edge_impulse/CMakeLists.txt

```diff
@@ -92,3 +92,7 @@ add_dependencies(edge_impulse
                  edge_impulse_project
                  edge_impulse_project_download
 )
+
+if(CONFIG_EI_NCS)
+  include(${CMAKE_CURRENT_LIST_DIR}/ei_ncs.cmake)
```

Comment:
I need a good reason why CMakeLists.txt content are moved into a `ei_ncs.cmake` file.

Reply:
I will move ei_ncs.cmake file to the CMakeLists.txt file